example_esphome_yaml/wroom_esp32s3_is2mic.yaml

```diff
@@ -0,0 +1,164 @@
+  friendly_name: wroom-esp32s3-yd23
+  platformio_options:
+    board_build.flash_mode: dio
+  on_boot:
+    - light.turn_on:
+        id: led_ww
+        blue: 100%
+        brightness: 60%
+        effect: fast pulse
+esp32:
+  board: esp32-s3-devkitc-1
+  variant: esp32s3
+  framework:
+    type: esp-idf
+
+    sdkconfig_options:
+      CONFIG_ESP32S3_DEFAULT_CPU_FREQ_240: "y"
+      CONFIG_ESP32S3_DATA_CACHE_64KB: "y"
+      CONFIG_ESP32S3_DATA_CACHE_LINE_64B: "y"
+      CONFIG_AUDIO_BOARD_CUSTOM: "y"
+   
+psram:
+  mode: octal
+  speed: 80MHz
+
+logger:
+
+ota:
+
+wifi:
+    ssid: !secret wifi_ssid
+    password: !secret wifi_password
+
+api:
+  on_client_connected:
+        then:
+          - delay: 50ms
+          - light.turn_off: led_ww
+          - micro_wake_word.start:
+  on_client_disconnected:
+        then:
+          - voice_assistant.stop: 
+
+button:
+  - platform: restart
+    name: "Restart"
+    id: but_rest
+
+switch:
+  - platform: template
+    id: mute
+    name: mute
+    optimistic: true
+    on_turn_on: 
+      - micro_wake_word.stop:
+      - voice_assistant.stop:
+      - light.turn_on:
+          id: led_ww           
+          red: 100%
+          green: 0%
+          blue: 0%
+          brightness: 60%
+          effect: fast pulse 
+      - delay: 2s
+      - light.turn_off:
+          id: led_ww
+      - light.turn_on:
+          id: led_ww           
+          red: 100%
+          green: 0%
+          blue: 0%
+          brightness: 30%
+    on_turn_off:
+      - micro_wake_word.start:
+      - light.turn_on:
+          id: led_ww           
+          red: 0%
+          green: 100%
+          blue: 0%
+          brightness: 60%
+          effect: fast pulse 
+      - delay: 2s
+      - light.turn_off:
+          id: led_ww 
+   
+light:
+  - platform: esp32_rmt_led_strip
+    id: led_ww
+    rgb_order: GRB
+    pin: GPIO48
+    num_leds: 1
+    rmt_channel: 0
+    chipset: ws2812
+    name: "on board light"
+    effects:
+      - pulse:
+      - pulse:
+          name: "Fast Pulse"
+          transition_length: 0.5s
+          update_interval: 0.5s
+          min_brightness: 0%
+          max_brightness: 100%
+          
+          
+ # Audio and Voice Assistant Config          
+i2s_audio:
+  i2s_lrclk_pin: GPIO11
+  i2s_bclk_pin: GPIO10
+
+microphone:
+  - platform: i2s_audio
+    id: va_mic
+    i2s_din_pin: GPIO09
+    adc_type: external
+    pdm: false
+    bits_per_sample: 16bit
+    channel: left
+
+    
+external_components:
+  - source: github://kahrendt/esphome-on-device-wake-word@dev
+    components: micro_wake_word
+    refresh: 0s 
+
+micro_wake_word:
+  on_wake_word_detected:
+    - voice_assistant.start:
+    - light.turn_on:
+        id: led_ww           
+        red: 30%
+        green: 30%
+        blue: 70%
+        brightness: 60%
+        effect: fast pulse 
+  model: github://kahrendt/esphome-on-device-wake-word/trained_models/okay_nabu/stream_state_internal_quantized.tflite
```

Comment:
Could you use the newest model version instead? It is a lot better:
``model: https://github.com/kahrendt/esphome-on-device-wake-word/raw/dev/trained_models/okay_nabu_v5_1/tflite_stream_state_internal_quant/stream_state_internal_quantize.tflite``

Reply:
have updated the yaml's in my repo - thanks so much for your awesome work